tests/unit/components/Records/Record/CollectionRecord/searchCollection.spec.js

```diff
@@ -88,7 +88,7 @@ Record.state.currentRecord["fairsharingRecord"] = {
     registry:'Collection'
 };
 
-describe("SearchCollection.vue", function(){
+describe("SearchOrganisationRecords.vue", function(){
```

Comment:
Was this change required? Seems the file original file name has not been changed.